packages/core/src/runtime/resolveStyleRules.test.ts

```diff
@@ -321,6 +321,72 @@ describe('resolveStyleRules', () => {
       `);
     });
 
+    it('handles layer queries', () => {
+      expect(
+        resolveStyleRules({
+          color: 'green',
+          '@layer color': { color: 'red' },
+        }),
+      ).toMatchInlineSnapshot(`
+        .fka9v86 {
+          color: green;
+        }
+        @layer color {
+          .fe3e8s9 {
+            color: red;
+          }
+        }
+      `);
+    });
+
```

Comment:
```suggestion

    it('handles layer queries with commas & dots', () => {
      expect(
        resolveStyleRules({
          '@layer theme, layout, utilities': { color: 'red' },
          '@layer framework.utilities': { color: 'red' },
        }),
      ).toMatchInlineSnapshot(`
        @layer theme,layout,utilities {
          .f16sacd2 {
            color: red;
          }
        }
        @layer framework.utilities {
          .faxdetk {
            color: red;
          }
        }
      `);
    });
```

Let's also add a test for commas & dots case.